app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -106,7 +107,7 @@ def sign_in_and_redirect
         controller = intake.controller_for_current_step
         to_path = controller.to_path_helper(
           action: controller.navigation_actions.first,
-          us_state: params[:us_state]
+          us_state: intake.state_code
```

Comment:
Pulling the state code from the intake rather than the params, so redirecting into the correct state if param is `us`